data_discovery_ai/pipeline.py

```diff
@@ -115,11 +115,13 @@ def prepare_sampleSet(self, raw_data: pd.DataFrame) -> pd.DataFrame:
         preprocessed_samples = preprocessor.sample_preprocessor(labelledDS, vocabs)
         sampleSet = preprocessor.calculate_embedding(preprocessed_samples)
 
+        # drop empty keywords rows
+        filtered_sampleSet = sampleSet[sampleSet["keywords"].apply(lambda x: x != [])]
+
         full_path = os.path.join(self.temp_dir, KEYWORD_SAMPLE_FILE)
 
-        preprocessor.save_to_file(sampleSet, full_path)
-        sampleSet = preprocessor.load_from_file(full_path)
-        return sampleSet
+        preprocessor.save_to_file(filtered_sampleSet, full_path)
```

Comment:
might have an info logging here to say successfully saved the file?

Reply:
`save_to_file` has the info logging already.